pkg/controllers/reconciler.go

```diff
@@ -113,57 +104,45 @@ type reconciler struct {
 	controller DevicePluginController
 	client.Client
 	scheme     *runtime.Scheme
+	dynamic    *dynamic.DynamicClient
```

Comment:
what's the `DynamicClient` about and where is it needed?

Reply:
It's used for fetching the untyped CR objects for cluster. I suppose it wouldn't have to be part of the reconciler struct.

Reply:
Need for dynamic was dropped.